Goal: Use the pager alternative as pager is PAGER is undefined

Fixes: #125603

Status wrt upstream: Debian specific

Author: Steve Langasek <vorlon@debian.org>

Note: 

Index: samba-3.0.25b/source/include/local.h
===================================================================
--- samba-3.0.25b.orig/source/include/local.h	2007-06-27 09:25:36.244206681 +0200
+++ samba-3.0.25b/source/include/local.h	2007-06-27 09:28:33.235816930 +0200
@@ -109,7 +109,7 @@
 /* the default pager to use for the client "more" command. Users can
    override this with the PAGER environment variable */
 #ifndef PAGER
-#define PAGER "more"
+#define PAGER "/usr/bin/pager"
 #endif
 
 /* the size of the uid cache used to reduce valid user checks */
